buildpacks/python.go

```diff
@@ -13,7 +13,7 @@ import (
 const (
 	anacondaToolVersion = "4.8.3"
 	// The version above needs a newer template
-	anacondaURLTemplate = "https://repo.continuum.io/miniconda/Miniconda{{.PyNum}}-py37_{{.Version}}-{{.OS}}-{{.Arch}}.{{.Extension}}"
+	anacondaURLTemplate = "https://repo.continuum.io/miniconda/Miniconda{{.PyNum}}-{{.Version}}-{{.OS}}-{{.Arch}}.{{.Extension}}"
```

Comment:
This will break our Python build pack. It needs that "-py37_", why did it remove it?

Reply:
staticcheck (correctly) reports that this constant isn't being used, so I'm replacing it with what is actually being used.

Reply:
Given that the old uppercase URL wasn't correct either I'm not 100% sure how this works now -- though maybe it doesn't (we should test this). There's a -latest, it might be easiest to just use that for now in the `anacondaToolVersion` (it seems their previous naming scheme was just the version up until 4.8)